packages/plugin-core/src/commands/pods/GoogleDocsExportPodCommand.ts

```diff
@@ -0,0 +1,206 @@
+        ? opts.exportScope
+        : await PodUIControls.promptForExportScope();
+
+    if (!exportScope) {
+      return;
+    }
+
+    const inputs = {
+      exportScope,
+      accessToken,
+      refreshToken,
+      ...opts,
+      podType: PodV2Types.GoogleDocsExportV2,
+      expirationTime,
+      connectionId,
+    };
+
+    // If this is not an already saved pod config, then prompt user whether they
+    // want to save as a new config or just run it one-time
+    if (!opts?.podId) {
+      const choice = await PodUIControls.promptToSaveInputChoicesAsNewConfig();
+
+      if (choice !== undefined && choice !== false) {
+        const configPath = ConfigFileUtils.genConfigFileV2({
+          fPath: path.join(
+            getExtension().podsDir,
+            "custom",
+            `config.${choice}.yml`
+          ),
+          configSchema: GoogleDocsExportPodV2.config(),
+          setProperties: _.merge(inputs, {
+            podId: choice,
+            podType: PodV2Types.GoogleDocsExportV2,
+            connectionId: inputs.connectionId,
+          }),
+        });
+
+        vscode.window
+          .showInformationMessage(
+            `Configuration saved to ${configPath}`,
+            "Open Config"
+          )
+          .then((selectedItem) => {
+            if (selectedItem) {
+              VSCodeUtils.openFileInEditor(vscode.Uri.file(configPath));
+            }
+          });
+      }
+    }
+
+    if (!isRunnableGoogleDocsV2PodConfig(inputs)) {
+      const id = await PodUIControls.promptToCreateNewServiceConfig(
+        ExternalService.GoogleDocs
+      );
+      await launchGoogleOAuthFlow(id);
+      vscode.window.showInformationMessage(
+        "Google OAuth is a beta feature. Please contact us at support@dendron.so or on Discord to first gain access. Then, try again and authenticate with Google on your browser to continue."
+      );
+      return;
+    } else {
+      return inputs;
+    }
+  }
+
+  async onExportComplete(opts: {
+    exportReturnValue: GoogleDocsExportReturnType;
+    payload: NoteProps;
+    config: RunnableGoogleDocsV2PodConfig;
+  }) {
+    const { exportReturnValue, payload } = opts;
+    if (ResponseUtil.hasError(exportReturnValue)) {
+      this.L.error(exportReturnValue.error);
+      window.showErrorMessage(
+        "Error while running Google Docs Export Pod: " + exportReturnValue
+      );
+      return;
+    }
+    const { documentId, revisionId } = exportReturnValue.data!;
+    if (documentId && revisionId) {
+      await this.updateNoteWithDocumentId({
+        documentId,
+        revisionId,
+        note: payload,
+      });
+    }
+    const gdocLink = `https://docs.google.com/document/d/${exportReturnValue.data?.documentId}`;
+    clipboard.writeText(gdocLink);
+    vscode.window.showInformationMessage(
+      `Finished running Google Docs export pod. The doc ${payload.fname} is created in your selected google docs account and link is copied to the clipboard.`
```

Comment:
let's shorten it a bit:

"The doc ${payload.fname} has been created. Its link has been copied to the clipboard."